src/mutable_data.rs

```diff
@@ -839,6 +839,10 @@ impl SeqEntryActions {
         Default::default()
     }
 
+    pub fn actions(&self) -> BTreeMap<Vec<u8>, SeqEntryAction> {
+        self.actions.clone()
```

Comment:
Maybe better to return a ref here and make a user choose to whether they want to clone it or not? i.e.
```rust
pub fn actions(&self) -> &BTreeMap<Vec<u8>, SeqEntryAction> {
    &self.actions
}
```